Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityMouseInputProfileInspector.cs

```diff
@@ -24,32 +26,29 @@ protected override void OnEnable()
 
         public override void OnInspectorGUI()
         {
-            RenderTitleDescriptionAndLogo(
-                "Mouse Input settings",
-                "Settings for mouse input in the editor.");
+            if (!RenderProfileHeader(ProfileTitle, ProfileDescription, BackProfileType.Input))
+            {
+                return;
+            }
 
+            // TODO: Troy
```

Comment:
>// TODO: Troy [](start = 12, length = 13)

Please clean up this TODO as well.

Reply:
This comment is on an outdated version of the file. It has already been removed